### ISSUE TITLE
Accessibility Bug

### ISSUE THREAD

**Issue by Peter-Peter254** [gertie-sheshe/nyenyeshi-dev#2]
Hi @gertie-sheshe I love your work. I have been following your work on a11y project of late and its compelling. I found an issue on this site [https://www.nyenyeshi.dev/](url). The exact issue is that the site violates Success Criterion 2.4.5. This WCAG criterion is related to multiple ways. 

Thanks.

**Comment by gertie-sheshe** [comment]
Hi @Peter-Peter254 thanks for this! Could you please provide more information and context on the issue you faced that violates 2.4.5? That will help me address the issue more concisely :) Thanks for flagging.

**Comment by Peter-Peter254** [comment]
Hi here is the link to this success criterion [https://www.w3.org/WAI/WCAG21/Understanding/multiple-ways.html](URL). The intent of this SC is to ensure all users can easily find a list of related pages within a website. WCAG recommends at least two ways. It could be a table of content, search, or repeating the links (Of related web pages in both the header) and the footer. Here is where the bug occurs.
![Screenshot_9](https://user-images.githubusercontent.com/105051105/220333415-f6effc44-0423-4bf3-a8a6-25e0a09a25f4.png)
Think of it this way:- A blind user is at the header section and wants to navigate the page "My Website". They would have to TAB through the entire page since only one way to find this page is provided. Suppose we had a search functionality or the same links repeated in the header. This would have been very easy for them. 

Thanks.

**Comment by gertie-sheshe** [comment]
Thanks again, Peter for flagging this concern.

1. All the links to other parts of the page are already provided at the top of the page for quick navigation to other pages. The docs read: "_For a three or four page site, with all pages linked from the home page, it may be sufficient simply to provide links from and to the home page_"
2. A blind user with assistive tech will be able to quickly navigate to the footer since it is a landmark on the page - they will not have to tab through the entire page to get to "My Website".
3. Having the same links repeated in the header poses a usability concern.

Could you kindly provide me with the tool you used to test for this? @Peter-Peter254

**Comment by Peter-Peter254** [comment]
1. My Bad 😒. Apologies for that, actually it would pass that  WCAG test case. references: (For a three or four page site, with all pages linked from the home page, it may be sufficient simply to provide links from and to the home page).  
2. From a compliance perspective, landmarks such as header and footer are more likely to fall under SC 1.3.1 . So yeah, checking on that the site passes the two Success Criterions. Apologies for bring so strict on that.
3. The point on the tools, such bugs cannot be reproduced using automated tools. Out of the 48 WCAG test cases, I prefer to do manual testing except for parsing. Then later confirm if there are any missed violations using automated tools.

if it's okay with you I can just nest another violations here. So that when this issue is closed, the bug will be fixed. We have undescriptive alt text for the profile pic as shown below.
![Screenshot_28](https://user-images.githubusercontent.com/105051105/220398744-3d1bee72-df08-4f4a-8bdd-2cfafb52ff01.png)
Screen reader users may never perceive that the avatar is your profile picture. I appreciate. Thanks @gertie-sheshe

**Comment by gertie-sheshe** [comment]
@Peter-Peter254 ah, this is an old site! I have plans to take it down soon - it has more than accessibility issues on it. Thanks for your contributions and comments!
